tutorials/02_ Run_a_workflow.md

```diff
@@ -16,9 +16,9 @@ In this tutorial, you will use the command line to view all available workflows
 
 # The quick tutorial
 
-> :warning: we assume you have VASP installed and that the `vasp` command is in the available path. In the future, we hope to update this tutorial with a workflow that doesn't require any extra installations. Until then, we apologize for the inconvenience. :cry:
+> :warning: we assume you have VASP installed and that the `vasp_std` command is in the available path. In the future, we hope to update this tutorial with a workflow that doesn't require VASP. Until then, we apologize for the inconvenience. :cry:
```

Comment:
The default command I use is `vasp > vasp.out`. Do you think I should change it to `vasp_std > vasp.out`?

Reply:
Yes.  Vasp installs by default as vasp_std.  I just preferred vasp so
renamed it on warwulf.